python/src/SymbolicFunction_doc.i.in

```diff
@@ -55,7 +55,6 @@ Available functions:
 
     Available operators:
 
-    - = (assignment)
```

Comment:
In exprtk it's ":=", do we want to keep it ?

Reply:
It is different with muparser and exprtk, thus I propose to drop it.  Advanced users will have to read exprtk documentation anyway.

Reply:
ok